storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -8023,9 +8023,7 @@ int main(int argc, char **argv) {
   }
 
 #ifndef __WIN__
-  if (xtrabackup_debug_sync) {
```

Comment:
if condition intentionally removed?

Reply:
yes, we set debug sync via debug_execute_if (--debug) parameter. We need to handle the cont signal.

Reply:
Ok, got it :+1: